larpandoradlcontent/LArDLContent.cc

```diff
@@ -0,0 +1,79 @@
+/**
+ *  @file   larpandoradlcontent/LArDLContent.cc
+ *
+ *  @brief  Factory implementations for content intended for use with particle flow reconstruction at liquid argon time projection chambers
+ *
+ *  $Log: $
+ */
+
+#include "Api/PandoraApi.h"
+
+#include "Pandora/Algorithm.h"
+#include "Pandora/AlgorithmTool.h"
+#include "Pandora/Pandora.h"
+
+#include "larpandoradlcontent/LArControlFlow/DLMasterAlgorithm.h"
+#include "larpandoradlcontent/LArDeepLearning/DeepLearningTrackShowerIdAlgorithm.h"
+
+#include "larpandoradlcontent/LArDLContent.h"
+
+#define LAR_DL_ALGORITHM_LIST(d)                                                                                                   \
+    d("LArDLMaster",                            DLMasterAlgorithm)                                                                 \
+    d("LArDeepLearningTrackShowerId",           DeepLearningTrackShowerIdAlgorithm)
```

Comment:
Right now, could leave this all super simple, with just the two factory registrations directly in the RegisterAlgorithms implementation. But, doing all the preprocessor stuff does make this more easily extensible as it grows.

Reply:
Yes, I thought keeping this consistent with the approach in LArContent was also worthwhile here.